Dockerfile

```diff
@@ -12,14 +12,17 @@ RUN apt-get update \
       supervisor \
       curl \
       bridge-utils \
-      net-tools
+      net-tools \
+      libboost-dev
 
 COPY debs /debs
-RUN dpkg -i /debs/libhiredis0.13_0.13.3-2_amd64.deb \
- && dpkg -i /debs/libhiredis-dev_0.13.3-2_amd64.deb \
- && dpkg -i /debs/libnl-3-200_3.2.27-2_amd64.deb \
- && dpkg -i /debs/libnl-genl-3-200_3.2.27-2_amd64.deb \
- && dpkg -i /debs/libnl-route-3-200_3.2.27-2_amd64.deb \
+RUN dpkg -i /debs/libhiredis0.14_0.14.0-3~bpo9+1_amd64.deb \
```

Comment:
I used these upgraded dependencies to build. Because Jenkins is deprecated, we can't use old jenkins artifacts.
I don't know how to get a clean build of restapi image.